__tests__/cron/dailyDigest.ts

```diff
@@ -348,7 +348,7 @@ describe('dailyDigest cron', () => {
       });
 
       it('should not schedule send time on Friday', async () => {
-        jest.setSystemTime(setDay(new Date(), DayOfWeek.Friday));
+        jest.setSystemTime(setHours(setDay(new Date(), DayOfWeek.Friday), 12));
```

Comment:
If it fails around midnight do we have a bug there then, or you think its just test setup stuff?

Reply:
I think it's just the tests, will double check a bit. Always hard when they're dependent on real time :ttiggered:

Reply:
yep, gets complicated

Reply:
@capJavert think it's just the tests, haven't actually encountered the issue in production